package.json

```diff
@@ -127,10 +127,12 @@
     "chai-as-promised": "^7.0.0",
     "chai-files": "^1.2.0",
     "co": "^4.6.0",
+    "detect-indent": "^5.0.0",
     "ember-cli-blueprint-test-helpers": "^0.18.0",
     "ember-cli-internal-test-helpers": "^0.9.0",
     "eslint-plugin-chai-expect": "^1.1.1",
     "eslint-plugin-mocha": "^4.8.0",
+    "fixturify": "^0.3.4",
```

Comment:
we should likely take advantage of `broccoli-test-helpers` which relies on [`fixturify`](https://github.com/broccolijs/broccoli-test-helper/blob/master/package.json#L39)

Reply:
I'm not actually sure which benefits does it offer here. Looks like `broccoli-test-helper` is intended to be used for testing of the build output diffs which is not related for testing blueprints output I think. 

Of course I may miss something because I've never used `broccoli-test-helper` before... I'd be glad to switch to `broccoli-test-helpers` if there is some reason.